index.html

```diff
@@ -5,22 +5,133 @@
+          <p class="dates3">2015</p></div>
+        <p class="prosumm">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tags">
+          <li class="tag1">html</li>
+          <li class="tag1">CSS</li>
+          <li class="tag1">Javascript</li>
+        </ul>
+        <div class="project">
+          <button type="button" class="probutton">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="photo"><img alt="twitter" src="Snapshoot_Portfolio_3.png"></div>
+        <h2>Multi-Post Stories</h2>
+        <div class="dates">
+          <p class="dates1">CANOPY</p>
+          <div class="circle"></div>
+          <p class="dates2">Back End Dev</p>
+          <div class="circle"></div>
+          <p class="dates3">2015</p></div>
+          <p class="prosumm">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tags">
+          <li class="tag1">html</li>
+          <li class="tag1">CSS</li>
+          <li class="tag1">Javascript</li>
+        </ul>
+        <div class="project">
+          <button type="button" class="probutton">See Project</button>
+        </div>
+      </div>
+      <div class="aboutMe">
+      <h2> <b>About Myself</b></h2>
+      <p class="prosumm">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <div class="connect"><a href="#">LET'S CONNECT</a></div>
+      <div class="connections">
+          <ul>
+            <li><button type="button"><img alt="twitter" src="Vector_1.png"></button></li>
+            <li><button type="button"><img alt="LinkedIn" src="Linkedin_icon.png"></button></li>
+            <li><button type="button"><img alt="github" src="Vector_2.png"></button></li>
+            <li><button type="button"><img alt="gmail" src="Vector_3.png"></button></li>
+            <li><button type="button"><img alt="github" src="Vector_4.png"></button></li>
+          </ul>
+          <br>
+        </div>
+        <div class="cv">
+          <button type="button" class="cvbutton">Get my resume</button>
+        </div>
+          <ul class="skillList">
+            <li class="works-item-lang"><p><b>Langguages</b></p><i class='fa fa-angle-down'></i></li>
+              <li class="Languages-item"><img alt="twitter" src="Javascript.png"><p class="lang">Javascript</p></li>
+              <li class="Languages-item"><img alt="twitter" src="HTML.png"><p class="lang">HTML</p></li>
+              <li class="Languages-item"><img alt="twitter" src="CSS.png"><p class="lang">CSS</p></li> 
+           <li class="works-item"><p><b>Frameworks</b></p> <i class='fa fa-angle-right'></i></li>
+           <li class="works-item"><p><b>Skills</b></p> <i class='fa fa-angle-right'></i></li>
+          </ul>
+    </div>
```

Comment:
- [ ] [REQUIRED] I see you have implemented most of the recommendations by the last reviewer, however, the about myself part still needs to be wrapped in a `section` tag and you need to fix the width/marginal error on the entire mobile design. Kindly refer to the Figma Template in order to get the proper margin values. This will help fix your design into the proper mobile size.
check the styling for your "container class" and your "wrapper class". The extra width seems to be coming from there because I see you have a fixed width and some padding as well.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/149540442-b1c72ec1-d576-462a-9ece-0321e35d2d47.png) | ![11](https://user-images.githubusercontent.com/58692590/149540478-43229896-c28a-492e-8385-786f2cb0a1bd.png)
### Same applies to other parts of the design
Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![2](https://user-images.githubusercontent.com/58692590/149542925-5b0ee9ff-beba-4ed4-8464-7abe7346268d.png) | ![22](https://user-images.githubusercontent.com/58692590/149542951-1b853bee-b48f-43b5-b6a7-02d933cc47a0.png)